.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,29 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '1.14'
+
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - run: npm run build --if-present
+    - run: npm test
+    - run: npm run test-conformance
```

Comment:
Does this script exist?

Reply:
Requires https://github.com/GoogleCloudPlatform/functions-framework-nodejs/pull/158